pdp/main.py

```diff
@@ -37,11 +37,11 @@ def initialize_frontend(global_config, use_analytics=False):
     static_app = static.Cling(resource_filename('pdp', 'static'))
 
     mounts = {
+        # Omit pcds frontend, because deprecated. (But we need the backend still.)
```

Comment:
Why do we still need the backend? Does the new frontend use it?

Reply:
It does. I don't want to remove it, although we do have a separate branch `pcds-only`, that is for the new frontend -- it's just the backend, as far as I know. James kindly built it for that purpose. In principle it _could_ be removed here since we are deploying for that purpose from the branch, not from the mainline. But that branch is intended eventually to be merged into the mainline, and I don't want to remove its basis before we do that.